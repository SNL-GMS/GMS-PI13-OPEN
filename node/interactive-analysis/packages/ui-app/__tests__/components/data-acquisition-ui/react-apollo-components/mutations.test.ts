import * as graphql from 'react-apollo';
import {
  graphqlAcknowledgeSohStatusMutation,
  graphqlQuietChannelMonitorStatusesMutation,
  graphqlSaveReferenceStationMutation,
  graphqlSaveStationGroupSohStatusMutation
} from '../../../../src/ts/components/data-acquisition-ui/react-apollo-components/mutations';
// tslint:disable-next-line:no-implicit-dependencies no-var-requires no-require-imports
// const Enzyme = require('enzyme');

// set up window alert and open so we don't see errors
(window as any).alert = jest.fn();
(window as any).open = jest.fn();

describe('mutations ', () => {
  const graphqlold = graphql.graphql;
  beforeAll(() => {
    (graphql as any).graphql = jest.fn() as any;
  });
  afterAll(() => {
    (graphql as any).graphql = graphqlold;
  });
  it('should be defined', () => {
    expect(graphqlSaveReferenceStationMutation).toBeDefined();
    expect(graphqlSaveStationGroupSohStatusMutation).toBeDefined();
    expect(graphqlAcknowledgeSohStatusMutation).toBeDefined();
    expect(graphqlQuietChannelMonitorStatusesMutation).toBeDefined();
  });
  it('should have graphqlSaveReferenceStationMutation', () => {
    graphqlSaveReferenceStationMutation();
    expect(graphql.graphql).toBeCalledTimes(1);
    expect((graphql.graphql as any).mock.calls[0]).toMatchSnapshot();
  });

  it('should have graphqlSaveStationGroupSohStatusMutation', () => {
    graphqlSaveStationGroupSohStatusMutation();
    expect(graphql.graphql).toBeCalledTimes(2);
    expect((graphql.graphql as any).mock.calls[1]).toMatchSnapshot();
  });

  it('should have graphqlAcknowledgeSohStatusMutation', () => {
    graphqlAcknowledgeSohStatusMutation();
    expect(graphql.graphql).toBeCalledTimes(3);
    expect((graphql.graphql as any).mock.calls[2]).toMatchSnapshot();
  });

  it('should have graphqlQuietChannelMonitorStatusesMutation', () => {
    graphqlQuietChannelMonitorStatusesMutation();
    expect(graphql.graphql).toBeCalledTimes(4);
    expect((graphql.graphql as any).mock.calls[3]).toMatchSnapshot();
  });
});
