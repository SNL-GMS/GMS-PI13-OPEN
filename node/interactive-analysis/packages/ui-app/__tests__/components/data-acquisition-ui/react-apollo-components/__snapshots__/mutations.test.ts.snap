// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mutations  should have graphqlAcknowledgeSohStatusMutation 1`] = `
Array [
  Object {
    "definitions": Array [
      Object {
        "directives": Array [],
        "kind": "OperationDefinition",
        "name": Object {
          "kind": "Name",
          "value": "acknowledgeSohStatus",
        },
        "operation": "mutation",
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [
                Object {
                  "kind": "Argument",
                  "name": Object {
                    "kind": "Name",
                    "value": "stationNames",
                  },
                  "value": Object {
                    "kind": "Variable",
                    "name": Object {
                      "kind": "Name",
                      "value": "stationNames",
                    },
                  },
                },
                Object {
                  "kind": "Argument",
                  "name": Object {
                    "kind": "Name",
                    "value": "comment",
                  },
                  "value": Object {
                    "kind": "Variable",
                    "name": Object {
                      "kind": "Name",
                      "value": "comment",
                    },
                  },
                },
              ],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "acknowledgeSohStatus",
              },
              "selectionSet": undefined,
            },
          ],
        },
        "variableDefinitions": Array [
          Object {
            "defaultValue": undefined,
            "directives": Array [],
            "kind": "VariableDefinition",
            "type": Object {
              "kind": "NonNullType",
              "type": Object {
                "kind": "ListType",
                "type": Object {
                  "kind": "NamedType",
                  "name": Object {
                    "kind": "Name",
                    "value": "String",
                  },
                },
              },
            },
            "variable": Object {
              "kind": "Variable",
              "name": Object {
                "kind": "Name",
                "value": "stationNames",
              },
            },
          },
          Object {
            "defaultValue": undefined,
            "directives": Array [],
            "kind": "VariableDefinition",
            "type": Object {
              "kind": "NamedType",
              "name": Object {
                "kind": "Name",
                "value": "String",
              },
            },
            "variable": Object {
              "kind": "Variable",
              "name": Object {
                "kind": "Name",
                "value": "comment",
              },
            },
          },
        ],
      },
    ],
    "kind": "Document",
    "loc": Object {
      "end": 156,
      "start": 0,
    },
  },
  Object {
    "name": "acknowledgeSohStatus",
    "withRef": false,
  },
]
`;

exports[`mutations  should have graphqlQuietChannelMonitorStatusesMutation 1`] = `
Array [
  Object {
    "definitions": Array [
      Object {
        "directives": Array [],
        "kind": "OperationDefinition",
        "name": Object {
          "kind": "Name",
          "value": "quietChannelMonitorStatuses",
        },
        "operation": "mutation",
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [
                Object {
                  "kind": "Argument",
                  "name": Object {
                    "kind": "Name",
                    "value": "channelMonitorsToQuiet",
                  },
                  "value": Object {
                    "kind": "Variable",
                    "name": Object {
                      "kind": "Name",
                      "value": "channelMonitorsToQuiet",
                    },
                  },
                },
              ],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "quietChannelMonitorStatuses",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "directives": Array [],
                    "kind": "FragmentSpread",
                    "name": Object {
                      "kind": "Name",
                      "value": "StationAndStationGroupSohFragment",
                    },
                  },
                ],
              },
            },
          ],
        },
        "variableDefinitions": Array [
          Object {
            "defaultValue": undefined,
            "directives": Array [],
            "kind": "VariableDefinition",
            "type": Object {
              "kind": "NonNullType",
              "type": Object {
                "kind": "ListType",
                "type": Object {
                  "kind": "NamedType",
                  "name": Object {
                    "kind": "Name",
                    "value": "ChannelMonitorInput",
                  },
                },
              },
            },
            "variable": Object {
              "kind": "Variable",
              "name": Object {
                "kind": "Name",
                "value": "channelMonitorsToQuiet",
              },
            },
          },
        ],
      },
      Object {
        "directives": Array [],
        "kind": "FragmentDefinition",
        "name": Object {
          "kind": "Name",
          "value": "StationAndStationGroupSohFragment",
        },
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "stationGroups",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "directives": Array [],
                    "kind": "FragmentSpread",
                    "name": Object {
                      "kind": "Name",
                      "value": "StationGroupSohStatusFragment",
                    },
                  },
                ],
              },
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "stationSoh",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "directives": Array [],
                    "kind": "FragmentSpread",
                    "name": Object {
                      "kind": "Name",
                      "value": "UiStationSohFragment",
                    },
                  },
                ],
              },
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "isUpdateResponse",
              },
              "selectionSet": undefined,
            },
          ],
        },
        "typeCondition": Object {
          "kind": "NamedType",
          "name": Object {
            "kind": "Name",
            "value": "StationAndStationGroupSoh",
          },
        },
      },
      Object {
        "directives": Array [],
        "kind": "FragmentDefinition",
        "name": Object {
          "kind": "Name",
          "value": "UiStationSohFragment",
        },
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "id",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "uuid",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "stationName",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "sohStatusSummary",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "needsAcknowledgement",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "needsAttention",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "time",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "statusContributors",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "directives": Array [],
                    "kind": "FragmentSpread",
                    "name": Object {
                      "kind": "Name",
                      "value": "SohContributorFragment",
                    },
                  },
                ],
              },
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "stationGroups",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "groupName",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "stationName",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "sohStationCapability",
                    },
                    "selectionSet": undefined,
                  },
                ],
              },
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "channelSohs",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "directives": Array [],
                    "kind": "FragmentSpread",
                    "name": Object {
                      "kind": "Name",
                      "value": "ChannelSohFragment",
                    },
                  },
                ],
              },
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "allStationAggregates",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "value",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "valuePresent",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "aggregateType",
                    },
                    "selectionSet": undefined,
                  },
                ],
              },
            },
          ],
        },
        "typeCondition": Object {
          "kind": "NamedType",
          "name": Object {
            "kind": "Name",
            "value": "UiStationSoh",
          },
        },
      },
      Object {
        "directives": Array [],
        "kind": "FragmentDefinition",
        "name": Object {
          "kind": "Name",
          "value": "SohContributorFragment",
        },
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "value",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "valuePresent",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "statusSummary",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "contributing",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "type",
              },
              "selectionSet": undefined,
            },
          ],
        },
        "typeCondition": Object {
          "kind": "NamedType",
          "name": Object {
            "kind": "Name",
            "value": "SohContributor",
          },
        },
      },
      Object {
        "directives": Array [],
        "kind": "FragmentDefinition",
        "name": Object {
          "kind": "Name",
          "value": "ChannelSohFragment",
        },
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "channelName",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "channelSohStatus",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "allSohMonitorValueAndStatuses",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "status",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "value",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "valuePresent",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "monitorType",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "hasUnacknowledgedChanges",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "contributing",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "quietUntilMs",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "quietDurationMs",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "thresholdBad",
                    },
                    "selectionSet": undefined,
                  },
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "thresholdMarginal",
                    },
                    "selectionSet": undefined,
                  },
                ],
              },
            },
          ],
        },
        "typeCondition": Object {
          "kind": "NamedType",
          "name": Object {
            "kind": "Name",
            "value": "ChannelSoh",
          },
        },
      },
      Object {
        "directives": Array [],
        "kind": "FragmentDefinition",
        "name": Object {
          "kind": "Name",
          "value": "StationGroupSohStatusFragment",
        },
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "stationGroupName",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "time",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "groupCapabilityStatus",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "id",
              },
              "selectionSet": undefined,
            },
            Object {
              "alias": undefined,
              "arguments": Array [],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "priority",
              },
              "selectionSet": undefined,
            },
          ],
        },
        "typeCondition": Object {
          "kind": "NamedType",
          "name": Object {
            "kind": "Name",
            "value": "StationGroupSohStatus",
          },
        },
      },
    ],
    "kind": "Document",
    "loc": Object {
      "end": 2426,
      "start": 0,
    },
  },
  Object {
    "name": "quietChannelMonitorStatuses",
    "withRef": false,
  },
]
`;

exports[`mutations  should have graphqlSaveReferenceStationMutation 1`] = `
Array [
  Object {
    "definitions": Array [
      Object {
        "directives": Array [],
        "kind": "OperationDefinition",
        "name": Object {
          "kind": "Name",
          "value": "saveReferenceStation",
        },
        "operation": "mutation",
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [
                Object {
                  "kind": "Argument",
                  "name": Object {
                    "kind": "Name",
                    "value": "input",
                  },
                  "value": Object {
                    "kind": "Variable",
                    "name": Object {
                      "kind": "Name",
                      "value": "input",
                    },
                  },
                },
              ],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "saveReferenceStation",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "result",
                    },
                    "selectionSet": undefined,
                  },
                ],
              },
            },
          ],
        },
        "variableDefinitions": Array [
          Object {
            "defaultValue": undefined,
            "directives": Array [],
            "kind": "VariableDefinition",
            "type": Object {
              "kind": "NonNullType",
              "type": Object {
                "kind": "NamedType",
                "name": Object {
                  "kind": "Name",
                  "value": "DataAcqReferenceStation",
                },
              },
            },
            "variable": Object {
              "kind": "Variable",
              "name": Object {
                "kind": "Name",
                "value": "input",
              },
            },
          },
        ],
      },
    ],
    "kind": "Document",
    "loc": Object {
      "end": 134,
      "start": 0,
    },
  },
  Object {
    "name": "saveReferenceStation",
  },
]
`;

exports[`mutations  should have graphqlSaveStationGroupSohStatusMutation 1`] = `
Array [
  Object {
    "definitions": Array [
      Object {
        "directives": Array [],
        "kind": "OperationDefinition",
        "name": Object {
          "kind": "Name",
          "value": "saveStationGroupSohStatus",
        },
        "operation": "mutation",
        "selectionSet": Object {
          "kind": "SelectionSet",
          "selections": Array [
            Object {
              "alias": undefined,
              "arguments": Array [
                Object {
                  "kind": "Argument",
                  "name": Object {
                    "kind": "Name",
                    "value": "input",
                  },
                  "value": Object {
                    "kind": "Variable",
                    "name": Object {
                      "kind": "Name",
                      "value": "input",
                    },
                  },
                },
              ],
              "directives": Array [],
              "kind": "Field",
              "name": Object {
                "kind": "Name",
                "value": "saveStationGroupSohStatus",
              },
              "selectionSet": Object {
                "kind": "SelectionSet",
                "selections": Array [
                  Object {
                    "alias": undefined,
                    "arguments": Array [],
                    "directives": Array [],
                    "kind": "Field",
                    "name": Object {
                      "kind": "Name",
                      "value": "result",
                    },
                    "selectionSet": undefined,
                  },
                ],
              },
            },
          ],
        },
        "variableDefinitions": Array [
          Object {
            "defaultValue": undefined,
            "directives": Array [],
            "kind": "VariableDefinition",
            "type": Object {
              "kind": "NonNullType",
              "type": Object {
                "kind": "ListType",
                "type": Object {
                  "kind": "NamedType",
                  "name": Object {
                    "kind": "Name",
                    "value": "StationGroupSohStatusInput",
                  },
                },
              },
            },
            "variable": Object {
              "kind": "Variable",
              "name": Object {
                "kind": "Name",
                "value": "input",
              },
            },
          },
        ],
      },
    ],
    "kind": "Document",
    "loc": Object {
      "end": 149,
      "start": 0,
    },
  },
  Object {
    "name": "saveStationGroupSohStatus",
    "withRef": false,
  },
]
`;
